Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/CorrelationAttributeInstance.java

```diff
@@ -111,7 +111,7 @@ public Boolean equals(CorrelationAttributeInstance otherInstance) {
     public String toString() {
         return this.getID()
                 + this.getCorrelationCase().getCaseUUID()
-                + this.getCorrelationDataSource().getName()
+                + this.getCorrelationDataSource().getDeviceID()
```

Comment:
Does this take into account the possibility that multiple data sources can have the same device id?

Reply:
I don't think they should.

Reply:
Okay, thanks. There may be future work to do here.